tests/TransactionTest.php

```diff
@@ -5,9 +5,9 @@
 
 class TransactionTest extends TestCase
 {
-    protected $insertData = ['name' => 'klinson', 'age' => 20, 'title' => 'admin'];
-    protected $originData = ['name' => 'users', 'age' => 20, 'title' => 'user'];
-    protected $connection = 'mongodb_repl';
+    protected array $insertData = ['name' => 'klinson', 'age' => 20, 'title' => 'admin'];
```

Comment:
I'm not sure what PHP version the library requires, as it's not noted in composer.json, but considering typed properties require PHP 7.4 we may want to undo this change.